### PR TITLE
Complete solution

### PR DESCRIPTION
**Pull request by StephDC**

### CODE REVIEW

File: urllibRequests.py [hunk 1]
```diff
@@ -1,12 +1,53 @@
...
+def get(urlStr,params={}):
+    reqdata = uReq.Request(urlStr)
     reqdata.add_header('User-Agent',
                        'VocabTool/0.2 (https://github.com/RihanWu/vocabtool)')
+    reqdata.add_header('Accept-Encoding','gzip, deflate')
     if params != {}:
-        reqdata.data = urllib.parse.urlencode(params).encode('ascii')
-    req = urllib.request.urlopen(reqdata)
-    return req.read()
+        reqdata.data = uPar.urlencode(params).encode('ascii')
```
Comment (RihanWu) [inline urllibRequests.py:37]: It seems that this is meant to be a POST request? Should we implement POST request along with GET request?

Comment (StephDC) [inline urllibRequests.py:37]: Yes you are right... I just realized that they would default to POST when data is present. I just need to manually set the method to GET in constructor. Change l32 

``` python
reqdata = uReq.Request(urlStr)
```

to

``` python
reqdata = uReq.Request(urlStr,method='GET')
```

would fix it.

**Comment by RihanWu** [comment]
I tested this module and find that it cannot handle http://worm.hanfucw.com/encodingText.txt

**Comment by StephDC** [comment]
Yes, because this file is using BIG5 encoding and it did not declare encoding in anywhere such as header. This module automatically fall back to UTF-8 to try to decode it, which might just fail.

**Comment by StephDC** [comment]
I just don't want to try to guess the encoding... And using chardet to solve the problem is doing no good in term of eliminating the 3rd party dependencies.

Requests can handle this because it is using chardet to guess the encoding when calling Response.apparent_encoding

**Pull request merged**